package.json

```diff
@@ -50,7 +50,7 @@
     "@babel/register": "^7.0.0",
     "babel-eslint": "^10.0.1",
     "babel-loader": "^8.0.2",
-    "babel-plugin-inline-react-svg": "^1.0.0",
+    "babel-plugin-inline-react-svg": "^1.1.0",
```

Comment:
@brandon-binary How about other vulnerability packages

Reply:
There shouldn't be other vulnerable packages after upgrading this.

Reply:
@brandon-binary After applying your changes in my local machine, `yarn audit` gave me 508 vulnerabilities packages 
![image](https://user-images.githubusercontent.com/16184705/62607180-3c22e780-b930-11e9-8ed3-809f059feab3.png)

Reply:
Did you reinstall npm_module?